bot.py

```diff
@@ -27,6 +27,42 @@
 )
 
 
+class Interests(discord.ui.Modal, title="Interests"):
+    interests = discord.ui.TextInput(
+        label="List out your interests.",
+        style=discord.TextStyle.long,
+        placeholder="Type your interests here...",
+        required=True,
+        max_length=300,
+    )
+
+    async def on_submit(self, interaction: discord.Interaction):
+        try:
+            print("Inputted interests: ", self.interests.value)
```

Comment:
Use `logging.info` instead of print, I've set it up to that logging also prints to the command line

Reply:
changed